data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/tracing/OpenTelemetryTracer.java

```diff
@@ -0,0 +1,208 @@
+package dev.knative.eventing.kafka.broker.core.tracing;
```

Comment:
This is a generic tracer based on open telemetry, that we can eventually upstream in `vertx` org or `opentelemetry/opentelemetry-java-instrumentation`.

Reply:
Yeah I guess we could contribute this back